src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -156,6 +156,8 @@
         </enum>
         <enum value="community-cloud">The community cloud deployment model as defined by <a href="https://doi.org/10.6028/NIST.SP.800-145">The NIST Definition of Cloud Computing</a>.
         </enum>
+        <enum value="hybrid-cloud"> The hybrid cloud deployment model, as defined by <a href="https://doi.org/10.6028/NIST.SP.800-145">The NIST Definition of Cloud Computing</a>, can be supported by selecting two or more of the existing deployment models.
+        </enum>
```

Comment:
Blocking: can we remove the leading space? More importantly, can you explain where the second half came from (" can be supported by selecting two or more of the existing deployment models") and how you would recommend that it would work? I would prefer it match the enumerations above and below it, but would like to understand if there was a reason this was added I may have missed.

```suggestion
        <enum value="hybrid-cloud"> The hybrid cloud deployment model, as defined by <a href="https://doi.org/10.6028/NIST.SP.800-145">.</enum>
```